spec/features/issue_trash_recover_spec.rb

```diff
@@ -15,6 +15,8 @@
     visit "/trash"
     click_link 'Recover'
 
+    sleep(1)
```

Comment:
@etdsoft I agree. Though, I would suggest we fix this in a separate and dedicated branch to keep things atomic.

@georgemillo on my machine, sometimes it passed and sometimes it didn't (which made debugging difficult). Also, is this line (https://github.com/dradis/dradis-ce/blob/master/spec/models/recoverable_revision_spec.rb#L95) the one you are referring to?